src/WebUI/Pods/PodsDetails.tsx

```diff
@@ -313,12 +313,25 @@ export class PodsDetails extends React.Component<IPodsDetailsProperties, IPodsDe
 
     // ToDO:: Handle GetImageDetails via ImageStore to avoid multiple calls to API from UI
     private _showImageDetails = (imageId: string) => {
-        const imageService = KubeFactory.getImageService();
-        imageService && imageService.getImageDetails(imageId).then(imageDetails => {
+        const showImageDetails = (imageDetails: IImageDetails): void => {
```

Comment:
we can remove the comment

// ToDO:: Handle GetImageDetails via ImageStore to avoid multiple calls to API from UI